doc/user_guide/configuration/all-options.rst

```diff
@@ -765,6 +765,13 @@ Standard Checkers
 **Default:**  ``7``
 
 
+--max-positional
+""""""""""""""""
+*Maximum number of positional arguments for function / method.*
+
+**Default:**  ``5``
```

Comment:
What should the default be? I've worked on projects where the limit was four, including self (three if static). 5 seems safe for now?

Reply:
Although I feel like `--max-positional-arguments` makes more sense.
Most people will use a config file so the added verbosity is a cost only paid once and then actually makes the config file a little bit more understandable.

Reply:
Yeah let's try to not annoy anyone too much. Afterall the pylint code base itself needs a 12 :D

Reply:
Yeah it shouldn't be higher than max total arguments, which defaults to 5 also.